xbmc/pvr/channels/PVRChannelGroups.cpp

```diff
@@ -313,8 +313,8 @@ bool CPVRChannelGroups::LoadFromDb()
               m_bRadio ? "radio" : "TV");
 
   // load all groups from the database
-  database->Get(*this);
-  CLog::LogFC(LOGDEBUG, LOGPVR, "Fetched {} {} groups from the database", m_groups.size(),
+  const int iLoaded = database->Get(*this);
```

Comment:
`const` has no effect here? Do you mean to just infer there is no need to change it, which is fine BTW.

Reply:
> infer there is no need to change it

This. As you know I'm a big fan of Item 21 in Scott Meyers' book Effective C++ ("Use const whenever possible").